src/Microsoft.ML.TimeSeries/ExtensionsCatalog.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Data;
+using Microsoft.ML.TimeSeriesProcessing;
+
+namespace Microsoft.ML
+{
+    public static class TimeSeriesCatalog
+    {
+        /// <summary>
+        /// Create a new instance of <see cref="IidChangePointEstimator"/>
+        /// </summary>
+        /// <param name="catalog">The transform's catalog.</param>
+        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.
+        /// Column is a vector of type double and size 4. The vector contains Alert, Raw Score, P-Value and Martingale score as first four values.</param>
```

Comment:
>Column [](start = 12, length = 6)

Which column?
Maybe say something like the "the output column will be a ...", can you also fix this in the other extensions?

Reply:
Its pretty clear we are referring to output column here. 

---
In reply to: [254484545](https://github.com/dotnet/machinelearning/pull/2344#discussion_r254484545) [](ancestors = 254484545)